src/app/_services/api/dashboard-new-cases.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { BaseService } from './base.service';
+import { CaseDataDto } from '../../_models/models';
+import * as constants from '../../app.constants';
+import { DashboardNewCasesSerializer } from '../../_serializers/dashboard-new-cases.serializer';
+import { HelperService } from '../helper.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DashboardNewCasesService extends BaseService<CaseDataDto> {
```

Comment:
are we going with the individual service for each new cases, new tests,etc.. I thought we were going to create a single DashbordService with different methods for the different endpoints

Reply:
Resolved.